mirdata/datasets/compmusic_jingju_acappella.py

```diff
@@ -0,0 +1,399 @@
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.audio_path = self.get_path("audio")
+
+        self.phoneme_path = self.get_path("phoneme")
+        self.phrase_char_path = self.get_path("phrase_char")
+        self.phrase_path = self.get_path("phrase")
+        self.syllable_path = self.get_path("syllable")
+
+    @core.cached_property
+    def phoneme(self):
+        return load_phonemes(self.phoneme_path)
+
+    @core.cached_property
+    def phrase(self):
+        return load_phrases(self.phrase_path)
+
+    @core.cached_property
+    def phrase_char(self):
+        return load_phrases_char(self.phrase_char_path)
+
+    @core.cached_property
+    def syllable(self):
+        return load_syllable(self.syllable_path)
+
+    @property
+    def work(self):
+        return self._track_metadata.get("work")
+
+    @property
+    def details(self):
+        return self._track_metadata.get("details")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path,
+            lyrics_data=[(self.phrase, "phrases"), (self.phrase_char, "phrases_char")],
+            event_data=[(self.phoneme, "phoneme"), (self.syllable, "syllable")],
+            metadata={
+                "work": self.work,
+                "details": self.details,
+            },
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load Jingju A Cappella Singing audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+    """
+    return librosa.load(fhandle, sr=44100, mono=True)
+
+
+@io.coerce_to_string_io
+def load_phonemes(fhandle: TextIO) -> annotations.EventData:
+    """Load phonemes
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to a phoneme annotation file
+
+    Returns:
+        EventData: phoneme annotation
+
+    """
+
+    start_times = []
+    end_times = []
+    events = []
+
+    reader = csv.reader(fhandle, delimiter="\t")
+    for line in reader:
+        start_times.append(float(line[0]))
+        end_times.append(float(line[1]))
+        events.append(str(line[2] if line[2] != "sil" else ""))
+
+    return annotations.EventData(np.array([start_times, end_times]).T, events)
```

Comment:
@genisplaja I guess you divided  phonemes and syllables `EventData` and the rest as `LyricData` differentiate annotations that have words from those that don't right? In that case, since the loaders seem exactly the same but I assume the annotation file is different, I'd suggest you group the loaders (i.e. instead of 4, have 2 loaders where the only difference is the annotation path). If that's not the case, ignore this comment :)

Reply:
Yeah, you're definitely right! I decided to load the Lyric annotation as `LyricsData` to be more understandable and to avoid confusion, phonemes and syllables are loaded as `EventData`. We discussed that with @nkundiushuti. I noticed now that I can definitely group the loaders...! In fact, the tests look the same as well. I'll do that.

Reply:
Well, now that I am looking deeply into it... For the Lyrics I think grouping is the way to go. However, I think phonemes and syllables is a more complex case because they are annotating different entities. If grouping, we should name the loading function as `load_phonemes_syllables`? Maybe it is interesting to have a function `load_syllables` and a function `load_phonemes` in case the user wants to load the annotations using them.